src/Storage/Storage.Management/Blob/GetAzureStorageBlobServiceProperties.cs

```diff
@@ -0,0 +1,109 @@
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.Management.Storage
+{
+    using Microsoft.Azure.Commands.Management.Storage.Models;
+    using Microsoft.Azure.Management.Storage;
+    using Microsoft.Azure.Management.Storage.Models;
+    using System;
+    using System.Collections.Generic;
+    using System.Management.Automation;
+    using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+    using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+
+    /// <summary>
+    /// Modify Azure Storage service properties
+    /// </summary>
+    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + StorageBlobServiceProperty, DefaultParameterSetName = AccountNameParameterSet), OutputType(typeof(PSBlobServiceProperties))]
+    public class GetAzStorageBlobServicePropertyCommand : StorageBlobBaseCmdlet
+    {
+
+        /// <summary>
+        /// AccountName Parameter Set
+        /// </summary>
+        private const string AccountNameParameterSet = "AccountName";
+
+        /// <summary>
+        /// Account object parameter set 
+        /// </summary>
+        private const string AccountObjectParameterSet = "AccountObject";
+
+        /// <summary>
+        /// BlobServiceProperties ResourceId  parameter set 
+        /// </summary>
+        private const string PropertiesResourceIdParameterSet = "BlobServicePropertiesResourceId";
+
+        [Parameter(
+          Position = 0,
+          Mandatory = true,
+          HelpMessage = "Resource Group Name.",
+         ParameterSetName = AccountNameParameterSet)]
+        [ResourceGroupCompleter]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(
+            Position = 1,
+            Mandatory = true,
+            HelpMessage = "Storage Account Name.",
+           ParameterSetName = AccountNameParameterSet)]
+        [ResourceNameCompleter("Microsoft.Storage/storageAccounts", nameof(ResourceGroupName))]
+        [Alias(AccountNameAlias)]
+        [ValidateNotNullOrEmpty]
+        public string StorageAccountName { get; set; }
+
+        [Parameter(Mandatory = true,
+            HelpMessage = "Storage account object",
+            ValueFromPipeline = true,
+            ParameterSetName = AccountObjectParameterSet)]
+        [ValidateNotNullOrEmpty]
+        public PSStorageAccount StorageAccount { get; set; }
+
+        [Parameter(
+            Position = 0,
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Blob Service Properties Resource Id.",
+           ParameterSetName = PropertiesResourceIdParameterSet)]
+        [ValidateNotNullOrEmpty]
+        public string ResourceId { get; set; }
+
+        [Parameter(Mandatory = false, HelpMessage = "Display ServiceProperties")]
+        public SwitchParameter PassThru { get; set; }
```

Comment:
Why does this have a passthru parameter?  The Get should always return the BlobServiceProperties.

Reply:
Removed. Thanks for catching it!